app/src/Event/Constraint/ValidCsvFile.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace Event\Constraint;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * @Annotation
+ */
+class ValidCsvFile extends Constraint
+{
+    public $groupname;
+    public $keyname;
+
+    public $message = 'The CSV must be formatted properly';
```

Comment:
Will this class check in the future whether there are the right fields in the right order?

Reply:
It can do whatever we want it to in the future.